frontend/src/components/NavBar.tsx

```diff
@@ -7,18 +7,24 @@ import Typography from '@mui/material/Typography';
 import Menu from '@mui/material/Menu';
 import MenuIcon from '@mui/icons-material/Menu';
 import Container from '@mui/material/Container';
-import Avatar from '@mui/material/Avatar';
 import Button from '@mui/material/Button';
 import Tooltip from '@mui/material/Tooltip';
 import MenuItem from '@mui/material/MenuItem';
 import logo from './../assets/logo.png';
+import AccountCircleRoundedIcon from '@mui/icons-material/AccountCircleRounded';
+import { NavLink } from "react-router-dom"; 
+const pages = ['Tours', 'Add Tour', 'My Tours'];
```

Comment:
Other code definitions and variables declaration must be below the library imports.

Also, Try to make library imports separate from directory imports. E:g., 
```
     import Menu from '@mui/material/Menu';
     import MenuIcon from '@mui/icons-material/Menu';
     
     import logo from './../assets/logo.png';
```